capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCapture.java

```diff
@@ -1092,6 +1102,9 @@ public CameraNavigationBarBottomAdapter getCameraNavigationBarBottomAdapter() {
             return cameraNavigationBarBottomAdapter;
         }
 
+        public NoResultsNavigationBarBottomAdapter getNoResultsNavigationBarBottomAdapter() {
```

Comment:
[Request] We also need a setter for this like we do for the other bottom navigation bars (e.g., `setOnboardingNavigationBarBottomAdapter()`).